pkg/queryfrontend/queryrange_codec.go

```diff
@@ -180,7 +180,7 @@ func parseDurationMillis(s string) (int64, error) {
 }
 
 func parseEnableDedupParam(s string) (bool, error) {
-	enableDeduplication := true
+	enableDeduplication := true // Deduplication is enabled by default.
```

Comment:
I think it would be nice to also update docs here https://thanos.io/tip/components/query.md/#deduplication saying that this option is enabled by default